web/src/components/Review.js

```diff
@@ -49,13 +49,14 @@ const Review = ({
             // If there's a delete click handler, add a remove button to the
             // header area and wire it up
             <Fragment>
-              |
+              {onEditClick && <strong>|</strong>}
               <Button
                 size="small"
                 variation="transparent"
                 onClick={onDeleteClick}
+                className="ds-u-color--error"
               >
-                Remove
+                Delete
```

Comment:
Do we use "Delete" throughout the app? We should aim for some consistency with the language

Reply:
This was part of another ticket I think that requested it be changed to delete, so I think that's what we are moving to.